l10n_be_vat_reports/wizard/partner_vat_list.py

```diff
@@ -41,7 +41,7 @@ def _compute_declarant_reference(self):
                 _("No VAT number associated with your company.")
             )
 
-        company_vat = company_vat.replace(" ", "").upper()
+        company_vat = company_vat.replace(" ", "").replace(".", "").upper()
```

Comment:
since this is done in 2 places, it should go into a function.

also, to avoid multiple replace operations, `re.sub()` can be used:

```python
company_vat = re.sub("[ .]", "", company_vat).upper()
```

or maybe use [fix_eu_vat_number()](https://github.com/OCA/OCB/blob/16.0/addons/base_vat/models/res_partner.py#L148) from the `base_vat` module directly?

Reply:
yes, using fix_eu_vat_number works, and it's a better solution, thanks :)
I port this change to the 14 branch